core/src/main/web/app/helpers/core.js

```diff
@@ -732,31 +751,41 @@
         if (!msgHeader) {
           msgHeader = "Question...";
         }
-        var close = function(result) {
-          if (result === "Yes") {
-            yesCB ? yesCB() : null;
-          } else if (result === "No") {
-            noCB ? noCB() : null;
-          } else { // cancel
-            cancelCB ? cancelCB() : null;
+        var callback = function(result) {
+          if (((result === "Yes") || (result == 0)) && yesCB) {
```

Comment:
I agree it looks funny, but I thought about it for a while without something better. Ill keep it in mind for a bit longer